content/docs/command-reference/add.md

```diff
@@ -85,6 +85,9 @@ but it can be used to track any large file or directory. We recommend using
 This way you bring data provenance and make your project
 [reproducible](/doc/command-reference/repro).
 
+To avoid tracking a file or directory, just add the corresponding
+[patterns](https://git-scm.com/docs/gitignore) in a `.dvcignore` file.
+
```

Comment:
OK but let's move this before "By default DVC tries to use reflinks" please.

Reply:
Hmmm also I don't think this is correct. `dvc add target` adds the target whether or not it's in .dvcignore. This only prevents files inside directories from being added accidentally. Please update the note.

Reply:
Let's also change it so you link to the Dvcignore user guide instead some Git web page.